xesmf/tests/test_frontend.py

```diff
@@ -566,6 +566,27 @@ def test_regrid_dataset():
     xr.testing.assert_identical(ds_result2, ds_result)
 
 
+@pytest.mark.parametrize('scheduler', dask_schedulers)
+def test_regrid_dataset_dask(request, scheduler):
+    scheduler = request.getfixturevalue(scheduler)
+    # xarray.Dataset containing dask array
+    regridder = xe.Regridder(ds_in, ds_out, 'conservative')
+
+    # `ds_out` already refers to output grid object
+    ds_result = regridder(ds_in.chunk())
+
+    # output should contain all data variables
+    assert set(ds_result.data_vars.keys()) == set(ds_in.data_vars.keys())
+    assert dask.is_dask_collection(ds_result)
+    assert ds_result.data.dtype == ds_in.data.dtype
+
+    ds_in_32 = ds_in.copy()
```

Comment:
although both are correct, I think people are more used to 4 bytes than 32 bits in the numerical types